recipes/ckanbatch-setup.rb

```diff
@@ -40,11 +40,35 @@
 	EOS
 end
 
-# Managed processes sometimes don't shut down properly on daemon stop,
-# leaving them 'orphaned' and resulting in duplicates.
-# Work around by issuing a stop command to the children first.
-execute "Stop children on supervisord stop" do
-	command <<-'SED'.strip + " /etc/init.d/supervisord"
-		sed -i 's/^\(\s*\)\(killproc\)/\1timeout 10s supervisorctl stop all || echo "WARNING: Unable to stop managed process(es) - check for orphans"; \2/'
-	SED
+if ::File.exist? "/etc/init.d/supervisord" then
```

Comment:
maybe extra comment what this is doing as first reading its not clear.

Reply:
Added.

Reply:
since f ::File.exist? "/etc/init.d/supervisord" is part of base ami image, this will work.